templates/emails/new_user.html

```diff
@@ -0,0 +1,25 @@
+<p>Hi there!</p>
+
+<p>Congrats! You're now a Django Girls organizer, so awesome! Here is access to your
+website:</p>
+
+<p>Your website address is http://djangogirls.org/{{ event.eventpage.url }}</p>
+
+<p>
+  Login here:<br />
+  http://djangogirls.org/admin/
+</p>
+
+<p>
+  Email: {{ user.email }}<br />
+  Password: {{ password }}
+</p>
+
+<p>Here you can find instructions on editing website: http://organize.djangogirls.org/website/</p>
+
+<p>We also invited you to Django Girls Slack, where we chat, communicate and meet each other!</p>
+
```

Comment:
This is absolutely awesome :heart_eyes: 
When I add someone to an existing event, I usually add info about the event email account: instructions on how to connect and that they should ask the main organizer to obtain the password.
It would be also cool to add the procedure for support ("You can contact hello@ if you need assistance but it’s usually faster to check our FAQ (link) or to post questions to our Google Group of Django Girls Organizers (link) or Slack channel (link) - there are more people that can help you! :)")